tests/kola/networking/default-network-behavior-change/test.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
+# No need to run on any other platform than QEMU.
+# kola: { "exclusive": false, "platforms": "qemu-unpriv" }
+set -xeuo pipefail
+
+# Since we depend so much on the default networking configurations let's
+# alert ourselves when any default networking configuration changes in
+# NetworkManager. This allows us to react and adjust to the changes
+# (if needed) instead of finding out later that problems were introduced.
+# some context in: https://github.com/coreos/fedora-coreos-tracker/issues/1000
+
+ok() {
+    echo "ok" "$@"
+}
+
+fatal() {
+    echo "$@" >&2
+    exit 1
+}
+
+# EXPECTED_INITRD_NETWORK_CFG1
+#   - used on FCOS 35+ releases
+EXPECTED_INITRD_NETWORK_CFG1="[connection]
+id=Wired Connection
+uuid=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+type=ethernet
+autoconnect-retries=1
+multi-connect=3
+permissions=
+
+[ethernet]
+mac-address-blacklist=
+
+[ipv4]
+dhcp-timeout=90
+dns-search=
+method=auto
+required-timeout=20000
+
+[ipv6]
+addr-gen-mode=eui64
+dhcp-timeout=90
+dns-search=
+method=auto
+
+[proxy]
+
+[user]
+org.freedesktop.NetworkManager.origin=nm-initrd-generator"
+# EXPECTED_INITRD_NETWORK_CFG2
+#   - used on all RHCOS releases
+#   - used on FCOS F34
+EXPECTED_INITRD_NETWORK_CFG2="[connection]
+id=Wired Connection
+uuid=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+type=ethernet
+autoconnect-retries=1
+multi-connect=3
+permissions=
+
+[ethernet]
+mac-address-blacklist=
+
+[ipv4]
+dhcp-timeout=90
+dns-search=
+method=auto
+
+[ipv6]
+addr-gen-mode=eui64
+dhcp-timeout=90
+dns-search=
+method=auto
+
+[proxy]"
+
+# EXPECTED_REALROOT_NETWORK_CFG1:
+#   - used on all FCOS/RHCOS releases
+EXPECTED_REALROOT_NETWORK_CFG1="[connection]
+id=Wired connection 1
+uuid=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+type=ethernet
+autoconnect-priority=-999
+interface-name=xxxx
+permissions=
+timestamp=xxxxxxxxxx
+
+[ethernet]
+mac-address-blacklist=
+
+[ipv4]
+dns-search=
+method=auto
+
+[ipv6]
+addr-gen-mode=stable-privacy
+dns-search=
+method=auto
+
+[proxy]
+
+[.nmmeta]
+nm-generated=true"
+
+# Function that will remove unique (per-run) data from a connection file
+normalize_connection_file() {
+    sed -e s/^uuid=.*$/uuid=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/ \
+        -e s/^timestamp=.*$/timestamp=xxxxxxxxxx/                 \
+        -e s/^interface-name=.*$/interface-name=xxxx/             \
+        $1
+}
+
+source /etc/os-release
+# All current FCOS releases use the same config
+if [ "$NAME" == "Fedora Linux" -o "$NAME" == "Fedora" ]; then
```

Comment:
Can't we do

```suggestion
if [ "$ID" == "Fedora" ]; then
```

?

Reply:
Sorry, that should be `fedora`, not `Fedora`.